client/src/locales/fr-FR.json

```diff
@@ -103,7 +103,12 @@
         "titles": {
             "workspaces": "Mes espaces",
             "settings": "Paramètres",
-            "devices": "Appareils"
+            "devices": "Appareils",
+            "activeUsers": "Utilisateurs actifs",
+            "revokedUsers": "Utilisateurs révoqués",
+            "invitations": "Invitations",
+            "storage": "Stockage",
+            "organization": "Informations"
```

Comment:
Same as above

Reply:
This was to avoid a case where a key named `information` could be used for multiple titles. I added subcategories instead.

Reply:
Seems legit